lib/NeuraDialect/Architecture/Architecture.cpp

```diff
@@ -344,10 +348,11 @@ void Architecture::createRegisterFileCluster(Tile *tile, int num_registers, int
   RegisterFileCluster *register_file_cluster = new RegisterFileCluster(tile->getId());
 
   // Creates registers as a register file.
+  int local_reg_id = 0;
   for (int file_idx = 0; file_idx < k_num_regfiles_per_cluster; ++file_idx) {
     RegisterFile *register_file = new RegisterFile(file_idx);
     for (int reg_idx = 0; reg_idx < k_num_regs_per_regfile; ++reg_idx) {
```

Comment:
`for (int reg_idx = 0; reg_idx < kNumRegsPerRegfile; ++reg_idx) {` -> `for (int reg = 0; reg < kNumRegsPerRegfile; ++reg) {`